static/js/Misc.jsx

```diff
@@ -3288,11 +3288,17 @@ const Autocompleter = ({getSuggestions, showSuggestionsOnSelect, inputPlaceholde
     )
 }
 
+const imgAltTextFactory = (caption) => {
+  const lang = Sefaria.interfaceLang === "hebrew" ? 'he' : 'en';
+  const fallBackMessage = lang === 'he' ? "Illustrative image" : "תמונה להמחשה";
+  const captionMessage = caption[lang];
+  if(captionMessage) {return captionMessage} else {return fallBackMessage}
```

Comment:
Use `Sefaria._v(caption)` to get the caption text translated. This function can be one line. I leave this as an exercise for the reader.